shell/platform/darwin/ios/framework/Headers/FlutterCodecs.h

```diff
@@ -37,6 +37,7 @@ FLUTTER_EXPORT
                                arguments:(id)arguments;
 @property(readonly, nonatomic) NSString* method;
 @property(readonly, nonatomic) id arguments;
+- (BOOL)isEqualToMethodCall:(FlutterMethodCall*)other;
```

Comment:
Yeah, I saw that later. I should have started a review instead of a single line comment. The `isEqualToXxx:` pattern is quite inconsistently followed. You can skip it if you are not using it yourself for a specific purpose.